### ISSUE TITLE
Why not use FAPE loss?

### ISSUE THREAD

**Issue by pengzhangzhi** [luost26/diffab#7]
Hi, I am wondering why you don't use fape loss to supervise the training. it is a more natural choice. It optimizes the rotation and translation predictions jointly.

**Comment by pengzhangzhi** [comment]
@luost26 Hi, shitong, I experiment the fape loss and observe that the loss and gradient go nan after one step of gradient descent. You may find out the similar issue and thus give up using fape in your model right? 
Would you like to talk about a bit of your experiences about this? 
I will keep this issue once I find out the solution or the reason.

**Comment by luost26** [comment]
We have many internal versions, including FAPE loss-based. All of them are trained successfully and we did not encounter NaN issues. Please double check.

**Comment by pengzhangzhi** [comment]
> 

Thanks for the information. I use OpenFold's implementation of FAPE. The problem maybe be it.  If fape works, why your paper version does not use it? Because the performance is lower than other versions?
